CONTRIBUTING.md

```diff
@@ -165,3 +166,7 @@ When reporting a bug please include the following key pieces of information:
 - The exact, minimal, steps needed to reproduce the issue.
   Submitting a 5 line script will get a much faster response from the team
   than one that's hundreds of lines long.
+  
+## Contributing a blog
+
+In order to contribute a blog to Istio, you'll need to open an [istio.io PR]((https://github.com/istio/istio.io/pulls]).  The proposed blog should be related to Istio and not vendor focused.  The blog must be reviewed and approved by at least one of the Istio Steering Committee members and at least one of the Documentation maintainers before it can be merged.
```

Comment:
I'll ask Maria to provide clarity on "not vendor focused"; I believe we do want to allow people to talk about how they use Istio in their commercial implementations or plug their third party platform into it, but within reason.

Reply:
We should clarify the requirements of an acceptable blog in the [contributing doc](https://istio.io/latest/about/contribute/add-content/#content-types), not here. This can then link to that doc.